tests/test_download.py

```diff
@@ -1,23 +1,19 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import sys
+
 import pytest
 from click.testing import CliRunner
 
 from mim.commands.download import download
 from mim.commands.install import cli as install
-from mim.commands.uninstall import cli as uninstall
-
-
-def setup_module():
-    runner = CliRunner()
-    result = runner.invoke(uninstall, ['mmcv-full', '--yes'])
-    assert result.exit_code == 0
-    result = runner.invoke(uninstall, ['mmcls', '--yes'])
-    assert result.exit_code == 0
 
 
 def test_download(tmp_path):
+    sys.path.append(str(tmp_path))
```

Comment:
Why do we need to append it to sys.path herer?

Reply:
The test cases of pytest are executed concurrently, but these test cases use the same python environment. I don't have a good way, and I am still trying to solve this problem.

Reply:
Updated